addons/cloud_storage_azure/utils/cloud_storage_azure_utils.py

```diff
@@ -373,7 +374,7 @@ def get_user_delegation_key(
     try:
         key_response = requests.post(key_request_url, data=key_data, headers=headers, timeout=5)
     except requests.exceptions.ConnectionError:
-        raise ValidationError("Failed to get user delegation key: the account name may be incorrect")
+        raise ValidationError(_("Failed to get user delegation key: the account name may be incorrect"))
```

Comment:
it's fine it there is really no language to be retrieved as this would actually mean that it's probably command line message so no big deal if it's untranslated. Also, picking the commit in the mentioned [PR](https://github.com/odoo/odoo/pull/174844) would include a lot of unrelated diffs + this would mean that this PR should wait till `Environment._` PR get merged. 

Also, it's fine if it remains untranslated after the PR is merged as it was actually untranslated before the PR.

Reply:
Ok. Someone will fix it in the future (may me in the other PR :) ).